src/main/main.c

```diff
@@ -163,6 +171,131 @@ void listen_to_buttons (void *pvParameters) {
     }
 }
 
+esp_bd_addr_t trusted_device;
+bool paired = false;
+
+static char *bda2str(esp_bd_addr_t bda, char *str, size_t size) {
```

Comment:
ikke tydeligt hvad den gør, eller hvorfor den returnere en reference til en karakter

Reply:
It is a default function from ESP IDF, to convert a bda to a MAC like string.